apps/src/templates/teacherDashboard/PrintCertificates.jsx

```diff
@@ -38,7 +38,7 @@ class PrintCertificates extends Component {
   };
 
   onClickPrintCerts = () => {
-    $.ajax(`/v2/sections/${this.props.sectionId}/students`).done(result => {
+    $.ajax(`/dashboardapi/sections/${this.props.sectionId}/students`).done(result => {
       const names = result.map(student => student.name);
       this.setState({names}, this.submitForm);
```

Comment:
Any testing for `PrintCertificates` or anything that confirms this action still works as expected?

Reply:
currently there is [this test](https://github.com/code-dot-org/code-dot-org/blob/91beaac010797bafbe183417119d1a04ab6a8748/apps/test/unit/templates/teacherDashboard/PrintCertificatesTest.js#L35-L55), which mocks the return value, so it doesn't test the endpoint itself. there are also [unit tests](https://github.com/code-dot-org/code-dot-org/blob/91beaac010797bafbe183417119d1a04ab6a8748/dashboard/test/controllers/api/v1/sections_students_controller_test.rb#L13-L37) on `GET /dashboardapi/sections/:section_id/students`, but no integration tests that i could find for `PrintCertificates`

Reply:
+1, please verify PrintCertificates some way before merging (not just in production after)